README.md

```diff
@@ -1,8 +1,8 @@
 ![Valyria Tear](https://raw.githubusercontent.com/ValyriaTear/ValyriaTear/master/data/boot_menu/valyria_logo.png)
 
-**Development blog:** http://valyriatear.blogspot.com  
-**Bug Tracker:** https://github.com/ValyriaTear/ValyriaTear/issues  
-**Forums:** http://forum.freegamedev.net/viewforum.php?f=76
+- [**Development blog:**](http://valyriatear.blogspot.com)
```

Comment:
Hey, since your turned the URL into anchor text, you can remove the colons ":" in the link text.
E.g.:
-**Development blog:** ...
into
-**Development blog** ...
Thanks!